docs/product/performance/requests.mdx

```diff
@@ -31,6 +31,14 @@ Availability of HTTP request monitoring depends on the SDK your application uses
 
 If the SDK you're using doesn't automatically instrument HTTP requests, you can instrument your application by following the <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink> instructions. Because all the concepts are the same, if your platform doesn't have dedicated custom instrumentation instructions, you can try adapting the code from another platform.
 
+### Span Eligibility
+
+If you are using <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation</PlatformLink>, query monitoring should work without any configuration. If you've manually instrumented Sentry, you'll need to make sure that your spans conform to our standards for the best experience:
+
+- The span `op` field is `http.client`
+- The span `description` is a string including the HTTP request method, and the full URL. e.g., "GET https://example.com/data.json?filter=all"
+- The span `data` is key-value mapping of span attributes. (e.g., {"http.query": "filter=all", "server.address": "prod-2.example.com"}). This optional but required for full experience.
+
```

Comment:
🤔 this section is redundant. The "Prerequisites and Limitations" section explains automatic instrumentation and what to do about custom instrumentation. Also, this condensed explanation of prerequisites doesn't list all the requirements _and_ duplicates them from another page.

I think a better change would be to link directly to https://develop.sentry.dev/sdk/performance/modules/requests/ somewhere in the "Prerequisites and Limitations" section, and leave this "Span Eligibility" section out.